addons/report_invoice_sysartec/views/report_papaerformat.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="paperformat_invoice_sysartec" model="report.paperformat">
```

Comment:
Why you create a paperformat?

Is not possible to use the current Letter paperformat?

Reply:
I created the paperformat because all the structure are in the page, and the space for header was there.